test/tests/disclosure_navigation_hybrid.js

```diff
@@ -82,7 +82,11 @@ ariaTest(
 
       if (links.length > 0) {
         await buttons[b].click();
+
         await links[0].click();
+        // Add a small delay here to ensure that the scroll to focus event
+        // has time to complete and doesn't interfere with the next assertion
+        await t.context.session.sleep(300);
```

Comment:
This test was giving intermittent failures. I determined that this was due to the scroll to focus event fired by the link click interfering with the assertion (`buttons[1]` always failed). This small delay fixes this issue.